app/src/composables/use-auto-fit-text.ts

```diff
@@ -0,0 +1,96 @@
+import { Ref } from 'vue';
+
+function extractValues(paddingString: string): { top: number; right: number; bottom: number; left: number } {
+	const paddingValues: number[] = paddingString
+		.split(' ')
+		.map((value) => parseFloat(value.trim()))
+		.filter((value) => !isNaN(value));
+
+	let top = 0,
+		right = 0,
+		bottom = 0,
+		left = 0;
+
+	switch (paddingValues.length) {
+		case 1:
+			// @ts-expect-error
+			top = right = bottom = left = paddingValues[0];
+			break;
+		case 2:
+			// @ts-expect-error
+			[top, right] = paddingValues;
+			bottom = top;
+			left = right;
+			break;
+		case 3:
+			// @ts-expect-error
+			[top, right, bottom] = paddingValues;
+			left = right;
+			break;
+		case 4:
+			// @ts-expect-error
+			[top, right, bottom, left] = paddingValues;
+			break;
```

Comment:
We should get rid of these "@ts-expect-error" comments, preferably by proper checks or using non-null assertion if not possible otherwise.